src/account/default.ts

```diff
@@ -116,6 +117,56 @@ export class Account extends Provider implements AccountInterface {
     };
   }
 
+  public async estimateAccountDeployFee(
```

Comment:
Maybe an opportunity to think, since methods have docs in the interface, does it make sense to have them here also, or remove all of them

Reply:
imo, we should only have docs in interface and not at the implementing methods